### PR TITLE
JTD 003 create tech table

### PR DESCRIPTION
**Pull request by dylanBkainos**

### CODE REVIEW

File: src/main/resources/V4__create_tech_table.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE technology (
+    techId int PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    tech_name varchar(30) NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/V4__create_tech_table.sql]: the naming convention has changed for this field?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Comment by lukemKainos** [comment]
Capitalise ticket name

File: src/main/resources/V4__create_tech_table.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE technology (
+    techId int PRIMARY KEY AUTO_INCREMENT NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/V4__create_tech_table.sql]: what is tech?

File: src/main/resources/V4__create_tech_table.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    techName varchar(30) NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/V4__create_tech_table.sql]: what is tech

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by lukemKainos — CHANGES_REQUESTED** [review]
You need to fix this pr name as mentioned

**Review by lukemKainos — APPROVED** [review]

**Review by dylanmckk — APPROVED** [review]

**Pull request merged**